compiler/rustc_macros/Cargo.toml

```diff
@@ -7,8 +7,10 @@ edition = "2021"
 proc-macro = true
 
 [dependencies]
+# FIXME: proc-macro2@1.0.72 doesn't enable the proc_macro_span
```

Comment:
We deliberately *don't* want it using these features. Just bless the worse errors and update.

dtolnay deactivating this just for us to not upgrade to it is very silly^^